pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -174,13 +174,13 @@ func (l *podNICImpl) PlugPhase1(vmi *v1.VirtualMachineInstance, iface *v1.Interf
 		return nil
 	}
 
-	cachedIface, err := l.loadCachedInterface()
+	cachedIface, err := l.getCachedInterface()
```

Comment:
At golang is idiomatic to remove the "get" prefix so this has to be "l.cachedInterface()" 

https://golang.org/doc/effective_go#Getters

Reply:
Done.